packages/datadog-plugin-grpc/test/integration-test/server.mjs

```diff
@@ -2,13 +2,12 @@ import 'dd-trace/init.js'
 import * as grpc from '@grpc/grpc-js'
 import * as protoLoader from '@grpc/proto-loader'
 import path from 'path'
-import getPort from 'get-port'
 
 const currentDirectoryPath = path.dirname(new URL(import.meta.url).pathname)
 const parentDirectoryPath = path.resolve(currentDirectoryPath, '..')
 
 let server
-const port = await getPort()
+let port
```

Comment:
Nit: Similar to my comment in your other PR (https://github.com/DataDog/dd-trace-js/pull/6543#discussion_r2387242214), I would add `= 0` here:

```suggestion
let port = 0
```